recipes/spacexr/meta.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: spacexr
+  version: 2.2.1
+
+source:
+  url: https://github.com/dmcable/spacexr/archive/refs/heads/master.zip
+  sha256: 05352900f07b2615cf6e81533697859311e73913646e24176b65c362b04fc67d
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("r-spacexr", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+
+requirements:
+  host:
+    - r-base
+    - r-remotes
+    - r-devtools
+    - r-readr
+    - r-ggplot2
+    - r-pals
+    - r-matrix
+    - r-doParallel
+    - r-foreach
+    - r-quadprog
+    - r-tibble
+    - r-dplyr
+    - r-reshape2
+    - r-knitr
+    - r-rmarkdown
+    - r-fields
+    - r-mgcv
+    - r-compquadform
+    - r-rfast
+    - r-locfdr
+    - r-metafor
+    - r-data.table
+  run:
+    - r-base
+    - r-remotes
+    - r-devtools
```

Comment:
are the devtools really needed during run-time?

Reply:
it is only needed for the installation

Reply:
```suggestion
```